src/scheduler/fifo.c

```diff
@@ -202,46 +199,20 @@ schedinit(int nthreads)
 #ifdef PYTHON
 	Py_NoSiteFlag = 1;
 	Py_FrozenFlag = 1;
-
-	/* Setting PYTHONHOME */
 	Py_IgnoreEnvironmentFlag = 1;
-        char pbs_python_home[MAXPATHLEN + 1];
-        memset((char *)pbs_python_home, '\0', MAXPATHLEN + 1);
-        snprintf(pbs_python_home, MAXPATHLEN, "%s/python",
-                pbs_conf.pbs_exec_path);
-        if (file_exists(pbs_python_home)) {
-                wchar_t tmp_pbs_python_home[MAXPATHLEN + 1];
-                wmemset((wchar_t *)tmp_pbs_python_home, '\0', MAXPATHLEN + 1);
-                mbstowcs(tmp_pbs_python_home, pbs_python_home, MAXPATHLEN + 1);
-                Py_SetPythonHome(tmp_pbs_python_home);
-        }
 
+	snprintf(python_binpath, MAXPATHLEN, "%s/python/bin/python3", pbs_conf.pbs_exec_path);
```

Comment:
Should we be calling something like get_py_progname() here? Looking at what was done in mom, I'm seeing some duplicate code.